src/reviews/reviews.ts

```diff
@@ -18,8 +20,9 @@ const wrapWithHtml = (reviews: string, doi: string, noHeader: boolean): string =
 
 export const generateReviewPage = (article: EnhancedArticle, noHeader: boolean): string => {
   if (article.reviews.length === 0) {
-    return wrapWithHtml('<li class="review-list__item"><article class="review-list-content">No reviews found</article></li>', article.doi, noHeader);
+    return wrapWithHtml(editorsAndReviewers(), '<li class="review-list__item"><article class="review-list-content">No reviews found</article></li>', article.doi, noHeader);
```

Comment:
If there are no reviews, are there any editors and reviewers? If not, probably just pass a blank string?

Reply:
No, I'd say not. Once the Reviewed Preprint (preprint + reviews) is available, displaying editor name(s) is fine, but we're not going to display reviewer names (for eLife ones at any rate). We wouldn't want editor or reviewer names displayed before reviews are available. Hope that helps!

Reply:
Thanks @acollings. Maybe we can base the decision to display the editors and reviewers on their inclusion in the docmap file which means we don't need to worry about the logic in this application.

I have added some commentary on this to the parent issue: https://github.com/elifesciences/enhanced-preprints/issues/119#issuecomment-1165283167